vision_agent/lmm/lmm.py

```diff
@@ -368,7 +368,7 @@ class AnthropicLMM(LMM):
     def __init__(
         self,
         api_key: Optional[str] = None,
-        model_name: str = "claude-3-5-sonnet-20240620",
+        model_name: str = "claude-3-5-sonnet-20241022",
```

Comment:
@dillonalaird can we please revert the claude version to the previous one and create a seperate PR with the updated version after getting some benchmark results?